cmp/tab/TabContainerModel.js

```diff
@@ -215,4 +208,41 @@ export class TabContainerModel {
 
         return null;
     }
+
+    setupStateProvider(persistWith) {
+        // Read state from provider -- fail gently
+        let state = null;
+        if (persistWith) {
```

Comment:
we know this is truthy -- only calling it when truthy in constructor

Reply:
Correct.   I have removed the if block.